public/locales/en/panel-keybinds.json

```
@@ -0,0 +1,20 @@
+{
+  "keyboard-view-tab-title": "Keyboard View",
+  "list-view-tab-title": "List View",
+  "keybind-info": {
+    "gui-path": "GUI Path",
+    "identifier": "Identifier",
+    "info": "Info",
+    "is-local": "Is Local"
+  },
+  "keyboard-layout": {
+    "selected-keybind-title": "Selected keybind",
+    "mapped-actions-title": "Mapped actions",
```

```suggestion
    "selected-keybind-title": "Selected Keybind",
    "mapped-actions-title": "Mapped Actions",
```
? To be consistent with the above titles